tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -101,24 +101,44 @@ isl::union_map partialSchedule(
 }
 
 namespace {
+/*
+ * If "domain" is any filter, then intersect "domain" with that filter.
+ */
+isl::union_set anyFilter(const ScheduleTree* node, isl::union_set domain) {
```

Comment:
How about calling this `applyAnyFilter` and swapping the order of arguments?  Makes the operation clear.  Maybe mark as `inline` as well.